core/src/main/java/org/openstack4j/openstack/compute/domain/ServerEvent.java

```diff
@@ -0,0 +1,89 @@
+package org.openstack4j.openstack.compute.domain;
+
+import java.util.Date;
+import java.util.List;
+
+import org.openstack4j.model.common.EventList;
+import org.openstack4j.model.compute.Event;
+import org.openstack4j.openstack.common.GenericEventsList;
+import org.openstack4j.openstack.common.ListResult;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+/**
+ * This is a model of a serverevent. It uses Jackson annotations for
+ * (de)serialization into JSON format
```

Comment:
This part does not really belong to Java Doc: "It uses Jackson annotations for (de)serialization into JSON format"